misc/alert_rules.json

```diff
@@ -30,5 +30,13 @@
   {
     "rule": "%ports.ifOperStatus = \"down\" && %ports.ifOperStatus_prev = \"up\" && %macros.device_up = \"1\"",
     "name": "Port status change from up to down"
+  },
+  {
+    "rule": "%ports.ifOutErrors_rate >= \"100\" || %ports.ifInErrors_rate >= \"100\"",
```

Comment:
It is true it may not but then I see these as more of examples of rules rather than absolutes. Before I had that rule I had no idea how to determine which ports were getting excessive errors.